modules/views/civicrm/civicrm_handler_relationship_participant.inc

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * This relationship handler is used when joining the civicrm_relationship table
+ * to the civicrm_contact table. This handler allows us to optionally add conditions
+ * to the join clause based on relationship_type_id and is_active.
+ */
+class civicrm_handler_relationship_participant extends views_handler_relationship {
+  static $participant_types;
+
+  /**
+   * Preload the list of relationship_types and store in the static variable
+   * $relationship_types
+   */
+  public function construct() {
+    parent::construct();
+
+    if (!civicrm_initialize()) {
+      return;
+    }
+    require_once 'CRM/Core/PseudoConstant.php';
```

Comment:
is this needed if civicrm is initialised?

Reply:
nope - but I discussed with @JohnFF & he's gonna use the api instead

Reply:
@seamuslee001 the CiviCRM initialise check is still needed most likely, because the API call comes next. I am now using the API, so the PseudConstant.php require is removed.